handlers/bootstrap.go

```diff
@@ -40,6 +40,8 @@ func ActivateMachine(event *events.Event, apiClient *client.RancherClient) error
 		return publishReply(reply, apiClient)
 	}
 
+	publishTransitioningReply("Installing Rancher agent...", event, apiClient)
```

Comment:
I like the idea of being somewhat consistent. Should we consider stripping the ... out of the messages from machine?

Otherwise, we can leave them in our messages. We'll just have to see how the look in the UI.